train_management/templates/train_management/dashboard.html

```diff
@@ -1,65 +1,10 @@
 {% extends "base.html" %}
 
 {% block content %}
-<h1>Some injected example text</h1>
+<h1>DVZO Dashboard</h1>
 
-<button id="button" class="btn btn-secondary">getSelections</button>
+<div class="container white-container">
+  Welcome to the DVZO Dashboard
```

Comment:
Add translation tag

Reply:
We iz ds würd blibe... aber so macht ds überhoupt ke sinn...